projects/unix/Makefile

```diff
@@ -131,7 +139,7 @@ OPTFLAGS ?= -O3 -flto
 WARNFLAGS ?= -Wall
 CFLAGS += -ffast-math -fno-strict-aliasing -fvisibility=hidden -I../../src -I../../src/asm_defines -DM64P_PARALLEL
 CXXFLAGS += -fvisibility-inlines-hidden
-LDLIBS +=  -lm
+LDLIBS +=  -lm -fPIC
```

Comment:
The "-fPIC" option doesn't belong in the LDLIBS variable. You should add it to LDFLAGS instead, and put it in a system or cpu-specific section